armi/reactor/reactors.py

```diff
@@ -210,6 +216,7 @@ def factory(cs, bp, geom: Optional[SystemLayoutInput] = None) -> Reactor:
     if cs[CONF_GEOM_FILE]:
         blueprints.migrate(bp, cs)
 
+    # For now, ARMI will create a default Spent Fuel Pool and add it to every reactor.
```

Comment:
I do not think this is something that should be necessary any more. As long as all the `excore` things are being written to the database, then building a SFP by default should not be needed. 

A default SFP should be added if the user has specified `trackAssems: True` in their blueprints. Otherwise, I do not think it needs to be or should be added.

Reply:
@keckler I have opened a separate ticket for this:  https://github.com/terrapower/armi/issues/1871

This is a separate ticket because that change will be bigger than this one. That change will require changes in downstream TP projects, and will generally more lines of code touched in ARMI than this one.  Now, it will also be a much easier change for me to make, but I'm just trying to keep this PR as small as I can.

TLDR; I agree!